### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by Auhan99**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿# DeepChem
+# DeepChem
```
Comment (kyscg) [inline README.md]: Is there a change here?

File: README.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ democratizes the use of deep-learning in drug discovery, materials science, quan
 * [tensorflow](https://www.tensorflow.org/)
 
 ### Soft Requirements
-DeepChem has a number of "soft" requirements. These are packages which are needed for various submodules of DeepChem but not for the package as a whole.
+DeepChem has some "soft" requirements. These are packages which are needed for various submodules of DeepChem but not for the package as a whole.
```
Comment (kyscg) [inline README.md]: I think the existing line was more grammatically correct

File: README.md [hunk 3]
```diff
@@ -68,7 +68,7 @@ ipython
 import deepchem as dc
 ```
 
-### Installing from source in a conda environment
+### Installing from the source in a conda environment
```
Comment (kyscg) [inline README.md]: 'Installing from source' is common terminology and I feel it should be left that way

File: README.md [hunk 4]
```diff
@@ -101,34 +101,34 @@ via this installation procedure.
...
 Two good tutorials to get started are [Graph Convolutional Networks](https://deepchem.io/docs/notebooks/graph_convolutional_networks_for_tox21.html) and [Multitask_Networks_on_MUV](https://deepchem.io/docs/notebooks/Multitask_Networks_on_MUV.html). Follow along with the tutorials to see how to predict properties on molecules using neural networks.
 
-Afterwards you can go through other [tutorials](https://deepchem.io/docs/notebooks/index.html), and look through our examples in the `examples` directory. To apply `deepchem` to a new problem, try starting from one of the existing examples or tutorials and modifying it step by step to work with your new use-case. If you have questions or comments you can raise them on our [gitter](https://gitter.im/deepchem/Lobby).
+Afterward you can go through other [tutorials](https://deepchem.io/docs/notebooks/index.html), and look through our examples in the `examples` directory. To apply `deepchem` to a new problem, try starting from one of the existing examples or tutorials and modifying it step by step to work with your new use-case. If you have questions or comments, you can raise them on our [gitter](https://gitter.im/deepchem/Lobby).
```
Comment (kyscg) [inline README.md]: Again, the original is grammatically correct

File: README.md (hunk 3 above)
Comment (Auhan99) [reply README.md]: I created a new pull request considering the suggestions

File: README.md (hunk 1 above)
Comment (Auhan99) [reply README.md]: no

**Pull request closed without merging**